build.sbt

```diff
@@ -217,6 +217,7 @@ lazy val codegenJava =
     .dependsOn(codegenCore % "compile->compile;test->test")
     .enablePlugins(PublishSonatype)
     .settings(
+      Test / fork := false, // needed to pass -D properties to ExampleSuite
```

Comment:
if we find it's worth, we can keep forking and pass the -D properties down the road. I didn't find it was worth though, but happy to change my mind.